lib/Importer/Caffe2.cpp

```diff
@@ -279,6 +279,38 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "FC") {
+    // Load the inputs:
+    auto *in = getOrCreateVariableByName(op.input(0));
+    // Load weights.
+    Tensor *w = getTensorByName(op.input(1));
+    Tensor *b = getTensorByName(op.input(2));
+
+    // Caffe2 stores the transposed W matrix. 
+    // In here we first coerce W into 2D tensor if necessary, 
+    // and then transpose W back.
+    Tensor wtag;
+    if (w->dims().size() != 2) {
+      size_t second = w->size() / b->size();
+      Tensor tmp(ElemKind::FloatTy, {b->size(), second});
```

Comment:
After reading comment here:
https://github.com/caffe2/caffe2/blob/0dd3284525079f3870df92f61bed3b94eb45ff53/caffe2/operators/fully_connected_op.cc#L167
I just think that we need to call `flattenCdr()`, like we do for `X` input upon lowering.

Also, turns out that they also have `axis` and `axis_w` arguments. While you are here, it may be good idea to assert that these are either not present, or are equal to 1.